backend/jest.config.cjs

```diff
@@ -3,6 +3,10 @@ module.exports = {
   transform: {
     "^.+\\.[t|j]sx?$": "babel-jest"
   },
+  // https://github.com/kulshekhar/ts-jest/issues/1057#issuecomment-1455530737
+  moduleNameMapper: {
+    "^(\\.\\.?\\/.+)\\.jsx?$": "$1"
```

Comment:
This is a workaround for an issue where jest can't seem to resolve ts module names. They should be transpiled to js and end in .js, but jest can't find those names unless the file extension is dropped.

Reply:
Huh interesting. This may have been the issue I was running into that made me give up testing. Nice work even if a workaround!